docker/hypervisor/networks/shared.xml

```diff
@@ -4,9 +4,9 @@
   <forward mode='nat'/>
   <bridge name='virbr1' stp='on' delay='0'/>
   <mac address='52:54:00:47:b3:70'/>
-  <ip address='192.168.123.1' netmask='255.255.255.0'>
+  <ip address='192.168.124.1' netmask='255.255.252.0'>
     <dhcp>
-      <range start='192.168.123.2' end='192.168.123.254'/>
+      <range start='192.168.127.33' end='192.168.127.254'/>
```

Comment:
From 192.168.124.1 to 192.168.127.33 are 799 addresses. To leave a 32 hosts outside of  DHCP, we would start the DHCP range to 192.168.124.33.

```suggestion
      <range start='192.168.124.33' end='192.168.127.254'/>
```